linker/shared/src/main/scala/org/scalajs/linker/backend/emitter/FunctionEmitter.scala

```diff
@@ -388,7 +389,7 @@ private[emitter] class FunctionEmitter(jsGen: JSGen) {
          * slower, but running GCC will take most of the time anyway in that case.
          */
         val globalRefs =
-          if (trackAllGlobalRefs) globalVarNames.toSet
+          if (config.trackAllGlobalRefs) globalVarNames.toSet
```

Comment:
Is this change necessary? There's an `import config._` at the top.

Reply:
No, it's not. I have found a bunch of others where the `import config._` makes the code shorter.

Reply:
> No, it's not

Consider reverting that change, then.

Reply:
Ugh... I thought I did... Updated.